tests/test_mu.py

```diff
@@ -411,7 +411,7 @@ def test_can_add_additional_files_while_open(self):
         archive.add_file(__file__)
         archive.close()
         filenames = archive.get_filenames()
-        self.assertTrue('test_mu.py' in filenames)
+        self.assertTrue(os.path.basename(__file__) in filenames)
```

Comment:
This looks good, but for completeness we should update the subsequent test as well.